pyanaconda/modules/boss/module_manager.py

```diff
@@ -107,6 +107,17 @@ def check_modules_availability(self):
 
         return True
 
+    def set_modules_locale(self, locale):
+        """Set locale of all modules.
+
+        :param str locale: locale to set
+        """
+        log.info("Setting locale of all modules to %s.", locale)
+        for observer in self.module_observers:
+            if not observer.is_service_available:
+                log.warning("%s is not available when setting locale", observer)
```

Comment:
You are missing `continue` here.

Reply:
Shame on me and thank you for this endless review, I think I should focus on a single thing at a time.